src/mono/wasi/Wasi.Build.Tests/BuildPublishTests.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using Wasm.Build.Tests;
+
+#nullable enable
+
+namespace Wasi.Build.Tests;
+
+public class BuildPublishTests : BuildTestBase
+{
+    public BuildPublishTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
+        : base(output, buildContext)
+    {
+    }
+
+    [Theory]
+    [InlineData("Release")]
+    public void BuildInLongPathSingleFileBundle(string config)
+    {
+        string id = $"{config}_{GetRandomId()}";
```

Comment:
does this test fail on CI without the fix ? (is the path long enough?)

Reply:
That failure I pasted in the description was from the CI. Yes, it fails nice.